src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,118 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double INCREASE_CAPACITY = 1.5;
+    private static final int ELEMENT_NOT_FOUND = -1;
+    private T[] elements;
+    private int size;
+
+    public ArrayList() {
+        elements = (T[]) new Object[DEFAULT_CAPACITY];
+    }
+
+    private T[] growIfArrayFull() {
```

Comment:
private methods should be placed after all public
```suggestion
    private void resize() {
```